### DISCUSSION TITLE
[Esame 9 settembre 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#610] [Esami M-Z]
![2](https://github.com/user-attachments/assets/0d8964d5-d312-4b88-83f3-95026c11b2fe)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/e4860fd2-aff5-490a-86bd-9b2936c07c1a)

**Comment by niccolo-fato** [comment]
<img width="1144" height="1269" alt="image" src="https://github.com/user-attachments/assets/356d00c9-8f8b-43f5-a006-8c59a3e54853" />
